docs/developer-docs/latest/setup-deployment-guides/configurations/optional/environment.md

```diff
@@ -1,11 +1,9 @@
 ---
 title: Environment configuration and variables - Strapi Developer Documentation
-description:
+description: In case you need specific static configurations for specific environments, and using environment variables becomes tedious, Strapi configurations can be created per environment.
```

Comment:
This description is too long, it would be nice to not exceed 160 characters :)

Reply:
```suggestion
description: In case you need specific static configurations for specific environments, configurations can be created per environment.
```